test/HealthChecks.Elasticsearch.Tests/HealthChecks.Elasticsearch.Tests.csproj

```diff
@@ -1,5 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup>
+    <TestTfmsInParallel>false</TestTfmsInParallel>

```

Comment:
just out of curiosity: why was this required?

Reply:
.NET 9 made a change to run tests in parallel by default: https://learn.microsoft.com/dotnet/core/whats-new/dotnet-9/sdk#run-tests-in-parallel.

This caused these tests to fail because when both the net8 and net9 tests are running at the same time, they both want to create a docker container with the same name:

```
error TESTERROR:
      HealthChecks.Elasticsearch.Tests.Functional.ElasticsearchAuthenticationTests.be_healthy_if_elasticsearch_is_using_valid_user_and_password (1ms):
       Error Message: Class fixture type 'HealthChecks.Elasticsearch.Tests.Fixtures.ElasticContainerFixture' threw in its constructor
      ---- Ductus.FluentDocker.Common.FluentDockerException : Could not create composite service with file(s) D:\git\AspNetCore.Diagnostics.HealthChec
      ks\test\HealthChecks.Elasticsearch.Tests\bin\Debug\net9.0/Resources/docker-compose.yml - result:  Network resources_default  Creating
       Network resources_default  Created
       Volume "resources_certs"  Creating
       Volume "resources_certs"  Created
       Volume "resources_esdata01"  Creating
       Volume "resources_esdata01"  Created
       Container resources-setup-1  Creating
      Error response from daemon: Conflict. The container name "/resources-setup-1" is already in use by container "aae4ab5800169ebc71eab2f914e418da6f
      156e283f373d34b44b431805fccca5". You have to remove (or rename) that container to be able to reuse that name.
      Stack Trace:

      ----- Inner Stack Trace -----
         at Ductus.FluentDocker.Services.Impl.DockerComposeCompositeService.Start()
         at Ductus.FluentDocker.Services.Impl.DockerComposeCompositeService.Ductus.FluentDocker.Services.ICompositeService.Start()
         at HealthChecks.Elasticsearch.Tests.Fixtures.ElasticContainerFixture..ctor() in D:\git\AspNetCore.Diagnostics.HealthChecks\test\HealthChecks.
      Elasticsearch.Tests\Fixtures\ElasticContainerFixture.cs:line 26
         at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
         at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
```

Telling `dotnet test` to run each TFM targetted test serially fixes this issue. I'll add a quick comment.

Reply:
Not related to this PR but i think we should change the tests to be able run on local. Currently tests only work when they run in CI.

Reply:
Some tests run locally. Some fail because the external container isn't set up - which you need to do manually.

I agree - it would be nice to be able to just run the tests locally. We could consider using TestContainers in a fixture, like we do in .NET Aspire.